parse_llms_txt.py

```
@@ -0,0 +1,265 @@
+
+            # Attempt to embed local document content if file exists
+            rel_path = item.get("url", "").lstrip("/")
+            file_path = os.path.join(root_dir, rel_path)
+            if os.path.isfile(file_path):
+                try:
+                    with open(file_path, "r", encoding="utf-8") as f:
+                        file_content = f.read()
+                    content_elem = ET.SubElement(doc_elem, "content")
+                    content_elem.text = file_content
+                except Exception:
+                    pass
```

Ruff (0.16.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)